components/p3a/scheduler_unittest.cc

```diff
@@ -38,24 +38,24 @@ class P3ASchedulerTest : public testing::Test {
 
   content::BrowserTaskEnvironment task_environment_;
   std::unique_ptr<Scheduler> scheduler_;
-  size_t upload_count_ = 0;
+  int upload_count_ = 0;
```

Comment:
i don't quite follow how does it help

Reply:
i'm not sure if it will either. i'm having trouble reproducing the issue given how rare it occurs. thought i'd give this a try just in case that it's data type related.